core/src/test/scala/org/apache/spark/SSLOptionsSuite.scala

```diff
@@ -109,7 +109,7 @@ class SSLOptionsSuite extends SparkFunSuite {
     val conf = new SparkConf
     val hadoopConf = new Configuration()
     conf.set("spark.ssl.enabled", "true")
-    conf.set("spark.ssl.ui.enabled", "false")
+    conf.set("spark.ssl.ui.enabled", "true")
```

Comment:
May I ask why we need this test case change in this PR?

Reply:
This change is required because the test case is validating the values set at sub settings (spark.ssl.ui.*) even though spark.ssl.ui.enabled is disabled. Before this change these values were set but now they won't be set. For this to work, spark.ssl.ui.enabled should be set to true.